src/main/resources/application.yml

```
@@ -20,4 +20,22 @@ spring:
+  secret: ${jwt.secret}
+
+token:
+  expire:
+    time:
+      access: ${token.expire.time.access}
+
+oauth:
+  kakao:
+    client-id: ${oauth.kakao.client-id}
+    redirect-uri: ${oauth.kakao.redirect-uri}
+    id-token-url: ${oauth.kakao.id-token-url}
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)